WordPress/Classes/ViewRelated/Jetpack Branding/JetpackBannerView.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+class JetpackBannerView: UIView {
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    func setup() {
+        guard AppConfiguration.isWordPress else {
+            return
+        }
+
+        translatesAutoresizingMaskIntoConstraints = false
+        backgroundColor = Appearance.jetpackBackgroundColor
+
+        let jetpackButton = makeJetpackButton()
+        addSubview(jetpackButton)
+
+        pinSubviewToAllEdges(jetpackButton, insets: UIEdgeInsets(top: 5, left: 0, bottom: 5, right: 0), priority: .required)
```

Comment:
Why do we need to add insets to the button? And if we do, could they be added to the `Appearance` enum?

Reply:
I added those insets to get close to the desired 44px height. I might try just making the button the full height though. 
Thanks for flagging.

Reply:
I've removed these, thanks!